src/render/render_common.hpp

```diff
@@ -24,9 +24,9 @@ namespace InternalConfig {
 inline constexpr uint32_t numFrames = 2;
 inline constexpr uint32_t initMaxTransforms = 100000;
 inline constexpr uint32_t initMaxMatIndices = 100000;
-inline constexpr uint32_t shadowMapSize = 4096;
+inline constexpr uint32_t shadowMapSize = 128;
```

Comment:
Can we bump this back up so shadows don't look horrible in the viewer?

Reply:
yes

Reply:
Again, doesn't seem resolved? I still see 128